airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalAttemptExecution.java

```diff
@@ -109,16 +116,7 @@ public OUTPUT get() {
       mdcSetter.accept(jobRoot);
 
       LOGGER.info("Executing worker wrapper. Airbyte version: {}", new EnvConfigs().getAirbyteVersionOrWarning());
-
-      // There are no shared volumes on Kube; only do this for Docker.
-      if (new EnvConfigs().getWorkerEnvironment().equals(WorkerEnvironment.DOCKER)) {
```

Comment:
any other pieces of the code that also access the workflow directory do so via helpers that will create any missing directories, so this is safe to delete.